### DISCUSSION TITLE
Sonoff 4CH Pro R3 not connecting to network  [Solved]

### DISCUSSION THREAD

**Discussion by jkimmel** [arendst/Tasmota#11377] [Q&A] [answered]
I flashed above device and could access the configuration menu.
Configured ssid and password and saved.
The device comes up the blue led blinking only twice.
I used the a scanned ssid and the password i used thousand times before.
But the device does not connect to my network.
An other observation: Pressing R1 the wifi led and the R1 led go on. The other R's no reaction

**Comment by barbudor** [comment, marked as answer]
Any special characters or spaces in your SSID or password ?
Do you have other ESP or Tasmota connected to your network ?

When not connecting to your network, the device should restart and present again the SSID AP tasmota_XXXX. Can you see this ?
Can you connect the USB/Serial and power the device from a safe 3.3v power and see the exact boot logs using a terminal such as Putty, Termite, Minicom ?
